x-pack/plugins/security_solution/public/common/utils/validators/index.test.ts

```diff
@@ -9,12 +9,40 @@ import { isUrlInvalid } from '.';
 
 describe('helpers', () => {
   describe('isUrlInvalid', () => {
-    test('verifies invalid url', () => {
+    test('should verify invalid url', () => {
       expect(isUrlInvalid('this is not a url')).toBeTruthy();
     });
 
+    test('should verify as invalid url without http(s):// prefix', () => {
+      expect(isUrlInvalid('www.thisIsNotValid.com')).toBeTruthy();
+    });
+
     test('verifies valid url', () => {
       expect(isUrlInvalid('https://www.elastic.co/')).toBeFalsy();
     });
+
+    test('should verify valid wwww such as 4 of them.', () => {
+      expect(isUrlInvalid('https://wwww.example.com')).toBeFalsy();
+    });
+
+    test('should validate characters such as %22 being part of a correct URL.', () => {
+      expect(isUrlInvalid('https://www.exam%22ple.com')).toBeFalsy();
+    });
+
+    test('should validate characters incorrectly such as ;', () => {
```

Comment:
Don't forget to double check my work :-) I think I meant to put `]` here.
```suggestion
    test('should validate characters incorrectly such as ]', () => {
```